src/HeapDump/GCHeapDumper.cs

```diff
@@ -1020,7 +1020,7 @@ private bool TryGetDotNetDump(int processID)
                     if (iCorDebug != null)
                     {
                         m_log.WriteLine("Attaching to silverlight process {0} from a {1} process.", processID,
-                            Environment.GetEnvironmentVariable("PROCESSOR_ARCHITECTURE"));
+                            Environment.Is64BitProcess ? ProcessorArchitecture.Amd64 : ProcessorArchitecture.X86);
```

Comment:
❓ Does silverlight work on ARM? Even if it does, do we need to handle it since HeapDump doesn't work on ARM?